GitCommands/Settings/AppSettings.cs

```diff
@@ -484,8 +484,8 @@ public static string IconStyle
 
         public static int AuthorImageSize
         {
-            get { return GetInt("authorimagesize", 80); }
-            set { SetInt("authorimagesize", value); }
+            get { return 80; }
+            set { /* deprecated */ }
```

Comment:
Thinking about it now - I probably should have... I can't remember why I didn't do it initially.
I think when I did this change I wasn't sure if I would update the commit info control as well.